Makefile.toml

```diff
@@ -30,7 +30,7 @@ command = "cargo"
 dependencies = ["format"]
 
 [tasks.build-variants]
-dependencies = ["build", "build-gateway", "build-driver", "build-simd"]
+dependencies = ["build", "build-gateway", "build-driver"]
```

Comment:
Can I ask why you've done this? `async-simd` is still supported as of 0.4.x.

Reply:
This makes some tests run when rebased on top of serenity-next. Probably due to this commit:

https://github.com/serenity-rs/songbird/commit/05cf36b040ff7c4c62be707dd83a2ec7962168b3#diff-9375fd04332c86472d7be397ef09428cb86babd8826880a5835bd1d1c1bdbc08L11

Could be removed if merging to the current branch I think.

Reply:
Yeah, the support is removed from `next` but still present here. Thanks for fixing that up.